### PR TITLE
Resiliency changes for gopowerscale

### PR DESCRIPTION
**Pull request by Sakshi-dell**
# PR Submission checklist

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/262 |

# Common PR Checklist:

- [x] Have you made sure that the code compiles?
- [x] Have you commented your code, particularly in hard-to-understand areas
- [x] Did you run tests in a real Kubernetes cluster?
- [x] Have you maintained backward compatibility

## Description of your changes:
Added a method IsIOinProgress() which gives the list of current nodes doing IO currently with the array.

### CODE REVIEW

File: api/v3/api_v3_cluster.go [hunk 1]
```diff
@@ -49,6 +49,15 @@ func GetIsiStats(
 	return resp, err
 }
 
+func IsIOinProgress(ctx context.Context,
+	client api.Client) (resp *ExportClientList, err error) {
+	err = client.Get(
+		ctx, string(platfromStatsPath), "summary/client", nil,
```
Comment (randeepsharma) [inline api/v3/api_v3_cluster.go:55]: could set this as constant and use it here, similar to platfromStatsPath

File: api/v3/api_v3_cluster.go [hunk 2]
```diff
@@ -49,6 +49,15 @@ func GetIsiStats(
 	return resp, err
 }
 
+func IsIOinProgress(ctx context.Context,
```
Comment (bpjain2004) [inline api/v3/api_v3_cluster.go:52]: rename this function to IsIOInProgress

File: cluster.go [hunk 3]
```diff
@@ -37,6 +38,18 @@ func (c *Client) GetStatistics(
 	return stats, nil
 }
 
+// IsIOinProgress checks whether a volume on a node has IO in progress
+func (c *Client) IsIOinProgress(
```
Comment (bpjain2004) [inline cluster.go:42]: rename this method to IsIOInProgress

File: cluster.go [hunk 4]
```diff
@@ -39,11 +39,11 @@ func (c *Client) GetStatistics(
 }
 
 // IsIOinProgress checks whether a volume on a node has IO in progress
-func (c *Client) IsIOinProgress(
+func (c *Client) IsIOInProgress(
 	ctx context.Context) (Clients, error) {
 
 	// query the volume without using the metadata parameter, use whether an error (typically, JSONError instance with "404 Not Found" status code) is returned to indicate whether the volume already exists.
-	stats, err := apiv3.IsIOinProgress(ctx, c.API)
+	stats, err := apiv3.IsIOInProgress(ctx, c.API)
```
Comment (bpjain2004) [inline cluster.go:46]: can we rename "stats" var to "clientList" ?

**Review by bpjain2004 — APPROVED** [review]

**Pull request merged**